MidasCivil_Adapter/CRUD/Read/Results/MeshResults.cs

```diff
@@ -98,12 +98,67 @@ private IEnumerable<IResult> ExtractMeshForce(List<int> ids, List<string> loadca
 
         private IEnumerable<IResult> ExtractMeshStress(List<int> ids, List<string> loadcaseIds, MeshResultLayer meshResultLayer)
         {
-            return null;
+            /***************************************************/
+            string filePath = directory + "\\Plate Stress(L).xls";
+            string csvPath = ExcelToCsv(filePath);
+            List<string> meshStressText = File.ReadAllLines(csvPath).ToList();
+            List<MeshStress> meshStresses = new List<MeshStress>();
+
+            List<string> meshStress = meshStressText[10].Split(',').ToList();
+            List<string> InitialmeshStress = meshStress;
+            meshStresses.Add(Convert.ToMeshStress(meshStress));
+
+            for (int i = 11; i < meshStressText.Count; i++)
+            {
+                meshStress = meshStressText[i].Split(',').ToList();
+                InitialmeshStress = meshStressText[i - 1].Split(',').ToList();
```

Comment:
You could declare and initialise the `initialMeshStress` at the end of the `i` for loop. 
- You could include your first conversion (`i = 10`) in the for loops
- You keep `initialMeshStress` in the loop so you don't store it